common/flavorization/00_CTP.txt

```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
 ## End of Dravidian_cultrue_group ##
 ###############################################
 
+###############################################
+## South_slavic_cultrue_group ##
```

Comment:
Again, culture not cultrue should be here.

Reply:
Fixed